lib/nudge_web/controllers/site_controller.ex

```diff
@@ -30,4 +30,17 @@ defmodule NudgeWeb.SiteController do
     site = Accounts.get_site!(id)
     render(conn, "show.html", site: site)
   end
+
+  def toggle(conn, %{"active" => active, "id" => id}) do
+    site = Accounts.get_site!(id)
+
+    case Accounts.update_site(site, %{active: active}) do
+      {:ok, _site} ->
+        redirect(conn, to: Routes.site_path(conn, :index))
+
+      {:error, %Ecto.Changeset{} = changeset} ->
+        put_flash(conn, :info, "Coudn't update.")
+        render(conn, "index.html", site: site, changeset: changeset)
```

Comment:
Shouldn't both these be piped with conn?

Reply:
Why do we need to pass changeset here?
We don't have changeset in index.html. just flash is enough